parts/linux/cloud-init/artifacts/components.json

```diff
@@ -237,7 +237,8 @@
       "downloadURL": "mcr.microsoft.com/aks/ip-masq-agent-v2:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v0.1.11"
+        "v0.1.11",
```

Comment:
keeping both until AKS has switched to v0.1.13 to reduce load on MCR (ip-masq-agnet runs on almost every k8s node in AKS)